PrivateHeaders/APPX/ZIP.h

```diff
@@ -327,6 +328,11 @@ namespace appx {
            << "xmlns=\"http://schemas.microsoft.com/appx/2010/blockmap\" "
            << "HashMethod=\"http://www.w3.org/2001/04/xmlenc#sha256\">";
         for (const ZIPFileEntry &entry : otherEntries) {
+            const std::string suffix = ".appx";
+            if (suffix.size() < entry.fileName.size() &&
+                    std::equal(suffix.rbegin(), suffix.rend(), entry.fileName.rbegin())) {
+                continue;
+            }
```

Comment:
We are excluding .appx files from the AppBloxMap.xml because .appxbundle files don't have them, it's more of an empirical conclusion/constraint rather than following a standard. 

As for what happens in the code path where we package appx bundles, we don't package APPX bundles into APPX bundles so this shouldn't be called. I can check to see what would happen (if at all possible) if we try this with the MakeAppx.exe Windows tool and we can incorporate that just in case if needed.